CorsixTH/CorsixTH.lua

```diff
@@ -44,7 +47,10 @@ else
     return ...
   end
 end
-function dofile(name)
+
+_G['corsixth'] = {}
+
+corsixth.require = function(name)
   if pathsep ~= "/" then
```

Comment:
This I would suggest to change to use "." as universal separator. That way it will behave the same way as Lua require works. Now this "/" was kinda weird to me. This of course means going through all the usages of `crosixth,require(.*\/.*)` and replace with `.`